src/plugins/data/server/search/search_service.test.ts

```diff
@@ -269,7 +268,7 @@ describe('Search service', () => {
 
       it('cancels a saved object and search ids', async () => {
         const mockMap = new Map<string, string>();
-        mockMap.set('abc', 'es');
+        mockMap.set('abc', 'ese');
```

Comment:
I think it is because default search strategy is now `ese`: 
https://github.com/elastic/kibana/pull/95811/files#diff-6ed2e0e44f6b08e16d11662deb4577612c758ae4b24e283c8f94d4566b67d2b6R275